### PR TITLE
402 userprofile shouted tab

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
402-userprofile-shouted-tab - text adapted for single or multiple. / css extended.
 
gif

### Issues
- relates 402

### Todo

- [X] None

### CODE REVIEW

File: webapp/pages/profile/_id/_slug.vue [hunk 1]
```diff
@@ -239,11 +241,16 @@
           <ds-flex-item class="profile-top-navigation">
             <ds-card class="ds-tab-nav">
               <ds-flex>
-                <ds-flex-item class="ds-tab-nav-item ds-tab-nav-item-active">
+
+                <ds-flex-item
+                    v-tooltip="{content: 'Your Posts', placement: 'right', delay: { show: 500 }}"
```
Comment (roschaefer) [inline webapp/pages/profile/_id/_slug.vue]: No untranslated strings, please.

File: webapp/pages/profile/_id/_slug.vue [hunk 2]
```diff
@@ -252,42 +259,62 @@
                     </no-ssr>
                   </ds-space>
                 </ds-flex-item>
-                <!--<ds-flex-item class="ds-tab-nav-item">
-                <ds-space margin="small">-->
-                <!-- TODO: find better solution for rendering errors -->
-                <!--
+                 
+                <ds-flex-item
+                  v-tooltip="{content: 'Your Comments', placement: 'right', delay: { show: 500 }}"
```
Comment (roschaefer) [inline webapp/pages/profile/_id/_slug.vue]: Same same :point_up:

File: webapp/pages/profile/_id/_slug.vue [hunk 3]
```diff
@@ -252,42 +259,62 @@
...
-                <!--<ds-flex-item class="ds-tab-nav-item">
-                <ds-space margin="small">-->
-                <!-- TODO: find better solution for rendering errors -->
-                <!--<no-ssr>
-                      <ds-number :label="$t('profile.shouted')">
-                        <hc-count-to slot="count" :end-val="user.shoutedCount"/>
+                </ds-flex-item>  
+                 
+                <ds-flex-item 
+                  v-tooltip="{content: 'Your Shouted', placement: 'left', delay: { show: 500 }}"
```
Comment (roschaefer) [inline webapp/pages/profile/_id/_slug.vue]: Same.

File: webapp/pages/profile/_id/_slug.vue [hunk 4]
```diff
@@ -252,42 +259,62 @@
...
             </ds-card>
           </ds-flex-item>
+
+      
+
           <ds-flex-item style="text-align: center">
             <ds-button
               v-if="myProfile"
-              v-tooltip="{content: 'Create a new Post', placement: 'left', delay: { show: 500 }}"
+              v-tooltip="{content: 'Create a new Post', placement: 'right', delay: { show: 500 }}"
```
Comment (roschaefer) [inline webapp/pages/profile/_id/_slug.vue]: ..translations :gb: : :de:

File: webapp/pages/profile/_id/_slug.vue [hunk 5]
```diff
@@ -393,6 +422,16 @@ export default {
...
+    },
+     tabActivity(str, e) {
+      console.log("tabActivity");
+      console.log(str);
+      console.log(e);
+     // document.removeClass('ds-tab-nav-item-active');
+     // this.class('.ds-tab-nav-item-active')
+     // $$('.ds-tab-nav-item-active').removeClass('ds-tab-nav-item-active');
+     // $$(activTab).class("")
+     // $$(this).attr('class', 'ds-tab-nav-item-active');
```
Comment (roschaefer) [inline webapp/pages/profile/_id/_slug.vue]: @ogerly this method does not do anything except `console.log`, please remove.

File: webapp/pages/profile/_id/_slug.vue (hunk 1 above)
Comment (ogerly) [reply webapp/pages/profile/_id/_slug.vue]: it should be everything in english. in my local version there is nothing more german.i pushed the latest one

Comment (ogerly) [reply webapp/pages/profile/_id/_slug.vue]: gif

**Comment by ulfgebhardt** [comment]
Does this PR also solves the Issue that the coresponding Filter is used for the Userposts?
-> Are the Posts the User shouted/commented displayed properly?

**Comment by ogerly** [comment]
> Behebt diese PR auch das Problem, dass der entsprechende Filter für die Userposts verwendet wird? 
> -> Werden die vom Benutzer geschrienen / kommentierten Posts richtig angezeigt?

No, not yet. It's pure css. I'm trying my hand at it right now.

**Comment by ogerly** [comment]
Sorry I'm still learning git!
this PR is closed.
continue here: https://github.com/Human-Connection/Human-Connection/pull/752

**Pull request closed without merging**